doc/user/content/guides/cdc-postgres.md

```diff
@@ -0,0 +1,260 @@
+
+### Deploy Debezium
+
+Debezium is deployed as a set of Kafka Connect-compatible connectors, so you first need to define a Postgres connector configuration and then start the connector by adding it to Kafka Connect.
+
+{{< note >}}
+Currently, Materialize only supports Avro-encoded Debezium records. If you’re interested in JSON support, please reach out in the community Slack or leave a comment in [this GitHub issue](https://github.com/MaterializeInc/materialize/issues/5231).
+{{</ note >}}
+
+1. Create a connector configuration file and save it as `register-postgres.json`:
+
+    ```json
+    {
+        "name": "your-connector",
+        "config": {
+            "connector.class": "io.debezium.connector.postgresql.PostgresConnector",
+            "tasks.max": "1",
+            "plugin.name":"pgoutput",
+            "database.hostname": "postgres",
+            "database.port": "5432",
+            "database.user": "postgres",
+            "database.password": "postgres",
+            "database.dbname" : "postgres",
+            "database.server.name": "pg_repl",
+            "table.include.list": "public.table1",
+            "publication.autocreate.mode":"filtered",
+            "key.converter": "io.confluent.connect.avro.AvroConverter",
+            "value.converter": "io.confluent.connect.avro.AvroConverter",
+            "value.converter.schemas.enable": false
+        }
+    }
+    ```
+
+    You can read more about each configuration property in the [Debezium documentation](https://debezium.io/documentation/reference/1.6/connectors/postgresql.html#postgresql-connector-properties). By default, the connector writes events for each table to a Kafka topic named `serverName.schemaName.tableName`.
+
+
+1. Start the Debezium Postgres connector using the configuration file:
+
+    ```bash
+    export CURRENT_HOST='<your-host>'
+
+    curl -i -X POST -H "Accept:application/json" -H  "Content-Type:application/json" \
+    http://$CURRENT_HOST:8083/connectors/ -d @register-postgres.json
+    ```
+
+1. Check that the connector is running:
+
+    ```bash
+    curl http://$CURRENT_HOST:8083/connectors/inventory-connector/status
+    ```
+
+    The first time it connects to a Postgres server, Debezium takes a [consistent snapshot](https://debezium.io/documentation/reference/1.6/connectors/postgresql.html#postgresql-snapshots) of the tables selected for replication; so, you should see that the pre-existing records in the replicated table are initially pushed into your Kafka topic:
+
+    ```bash
+    /usr/bin/kafka-avro-console-consumer \
+      --bootstrap-server kafka:9092 \
+      --from-beginning \
+      --topic pg_repl.public.table1
+    ```
+
+### Create a source
```

Comment:
Again, we should signal the shift from working with Postgres/Debezium to working with Materialize here.